sdk/appconfiguration/Azure.Data.AppConfiguration/samples/Sample12_GetSettingsUsingTags.md

```diff
@@ -0,0 +1,72 @@
+# Asynchronously List Settings With Tags
+
+This sample demonstrates how to send requests to the Azure App Configuration service asynchronously. It also shows how to create configuration settings with tags, and provides an example scenario where tags are used to retrieve the settings.  To get started, you'll need a connection string to Azure App Configuration. See the [README](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/appconfiguration/Azure.Data.AppConfiguration/README.md) for links and instructions.
+
+ ## Create a ConfigurationClient
+
+To interact with Azure App Configuration, you need to instantiate a `ConfigurationClient`. You can use either an endpoint URL and a [`TokenCredential`](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/identity/Azure.Identity/README.md#credentials) or a connection string.
+
+For the sample below, you can set `connectionString` in an environment variable, a configuration setting, or any way that works for your application. The connection string is available from the App Configuration Access Keys view in the Azure Portal.
+
+```C# Snippet:AzConfigSample12_CreateConfigurationClient
+var client = new ConfigurationClient(connectionString);
+```
+
+## Asynchronously create configuration settings
```

Comment:
why is "asynchronously" stressed so much here?

Reply:
I used an existing sample's text as a template. I'll update the text for this new sample.